src/zenml/cli/hub.py

```diff
@@ -995,10 +974,68 @@ def _ask_for_tags() -> List[str]:
             tags.append(tag)
 
 
+@hub.command("submit-batch")
+@click.argument(
+    "config", type=click.Path(exists=True, dir_okay=False), required=True
+)
+def batch_submit(config: str) -> None:
+    """Batch submit plugins to the ZenML Hub.
+
+    WARNING: This command is intended for advanced users only. It does not
+    perform any client-side validation which might lead to server-side HTTP
+    errors that are hard to debug if the config file you specify is invalid or
+    contains invalid plugin definitions. When in doubt, better use the `zenml
+    hub submit` command instead.
+
+    Args:
+        config: Path to the config file. The config file is expected to be a
+            list of plugin definitions. Each plugin definition must be a dict
+            with keys and values matching the fields of `HubPluginRequestModel`:
+            ```yaml
+            - name: str
+              version: str
+              release_notes: str
+              description: str
+              repository_url: str
+              repository_subdirectory: str
+              repository_branch: str
+              repository_commit: str
+              logo_url: str
+              tags:
+                - str
+                - ...
+            - ...
+            ```
+    """
+    from pydantic import ValidationError
+
+    from zenml.utils.yaml_utils import read_yaml
+
+    config = read_yaml(config)
+    client = HubClient()
+    declare(f"Submitting {len(config)} plugins to the hub...")
```

Comment:
I think we should move this below the check that makes sure that `config` is a dict, otherwise the `len(...)` call might lead to some hard to understand error message?

Reply:
good point 👍